FreeRTOS-Plus/Demo/FreeRTOS_Plus_WolfSSL_FIPS_Ready_Windows_Simulator/README_wolfSSL_FIPS_ready.md

```diff
@@ -36,6 +36,19 @@ All required settings have been set in the user_settings.h header file included
 2. Run the RTOSDemo.exe 
 
 You will see a console that pops up, and it shows output like the following:
+<<<<<<< HEAD
```

Comment:
This looks like a merge error that did not cleaned up

Reply:
Cleaned up the conflict marker section.